packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -730,6 +730,29 @@ Fetches a user from your Okta organization
 - [Get User with Login](#get-user-with-login)
 - [Get User with Login Shortname](#get-user-with-login-shortname)
 
+##### Content-Type Parameters
+
+All variants of this API support a performance optimization.  Complex DelAuth configurations may degrade performance when fetching specific parts of the response.  Passing an optional Content-Type parameter can omit these parts, bypassing the bottleneck.
+
+| Parameter     | Description                                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| okta-response | A comma-separated list of omit options (optionally surrounded in quotes) each specifying a specific part of the resonse to be omitted |
+
+| Omit Option | Description                                                                                                                     |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| omitCredentials | Omits the credentials subobject from the response.  |
+| omitCredentialsLinks | Response omits the following HAL links:  Change Password, Change Recovery Question, Forgot Password, Reset Password, Reset Factors, Unlock |
+| omitTransitioningToStatus | Omits the transitioningToStatus field from the response. |
```

Comment:
Why not use the same format for each Description?  E.g. middle could be "Omits the following HAL links from the response: ...", or first could be "Response omits the credentials subobject."
nit: Middle item should end with period after list of links.

Reply:
Slightly more terse to avoid one this one row ballooning in length.